src/pages/SignUp/SignUp.jsx

```diff
@@ -22,6 +24,7 @@ export const SignUp = () => {
     try {
       const data = await registerAPICall(registerInputState);
       authStateDispatch({ type: "LOGIN", payload: data.encodedToken });
+      userDataDispatch({ type: "LOGIN", payload: data.createdUser });
```

Comment:
here you are using LOGIN constants right so you can create one constant file and you store all constans there and get from there what ever constant you want